src/test/java/org/fundacionjala/coding/yerel/PersistenteInseparableTest.java

```diff
@@ -0,0 +1,53 @@
+package org.fundacionjala.coding.yerel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * this is a test PersistenteInseparableTest.
+ */
+public class PersistenteInseparableTest {
+
+    private static final int TESTNUMBER_999 = 999;
+    private static final int TESTNUMBER_39 = 39;
+    private static final int TESTNUMBER_4 = 4;
+    private static final int TESTEXPECTED_3 = 3;
+    private static final int TESTEXPECTED_4 = 4;
+    private static final int TESTEXPECTED_0 = 0;
```

Comment:
Don't create constants
Please create final local variables in the method as actualResult and expectedResult